mlrun/datastore/azure_blob.py

```diff
@@ -32,44 +33,143 @@ class AzureBlobStore(DataStore):
 
     def __init__(self, parent, schema, name, endpoint="", secrets: dict = None):
         super().__init__(parent, name, schema, endpoint, secrets=secrets)
+        self._service_client = None
+        self._storage_options = None
+        self.max_concurrency = 100
+        self.max_blocksize = 1024 * 1024 * 4
+        self.max_single_put_size = 1024 * 1024 * 8  # for BlobServiceClient only
```

Comment:
If the intention is not to allow for overriding them, they should be moved to the class level.